.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,26 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
```

Comment:
Does this also install our own dependencies such as `requests`? Or, they don't even need to be installed to build the distribution?

Reply:
I'm not sure :thinking: I haven't seen any docs saying I need to install packages ahead of time, my guess is that the upload / build doesn't require installing dependencies, but let me see if I can find some documentation that explicitly mentions this.

Reply:
Ok so from quickly testing it out on the legacy pypi server, it seems having the dependencies installed locally is not required for packaging. That's pretty neat!